File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/21a_compito_java_20180406/Osteria.py
from Ristorante import Ristorante


class Osteria(Ristorante):
    def __init__(self, codice, tipo, nome, posti, superficie, bagno_disabili):
        super().__init__(codice, tipo, nome)
        self._posti = posti
        self._superficie = superficie
        self._bagno_disabili = bagno_disabili

    def __str__(self):
        return super().__str__() + "\t-\t-\t" + str(self._posti) + "\t" + str(self._superficie) + "\t" + str(self._bagno_disabili)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/21_compito_java_2018_04_06a/Osteria.java
public class Osteria extends Ristorante {
	private int postiASedere;
	private double superficie;
	private boolean bagnoDisabili;

	public Osteria(int codiceRistorante, String tipoRistorante, String nomeRistorante, int postiASedere,
			double superficie, boolean bagnoDisabili) {
		super(codiceRistorante, tipoRistorante, nomeRistorante);
		this.postiASedere = postiASedere;
		this.superficie = superficie;
		this.bagnoDisabili = bagnoDisabili;
	}

	public String toString() {
		return nomeRistorante + "\t" + codiceRistorante + "\t" + tipoRistorante + "\t-\t-\t" + postiASedere + "\t"
				+ superficie + "\t" + bagnoDisabili;
	}
}